RESideMenu/RESideMenu.m

```diff
@@ -649,9 +667,11 @@ - (void)panGestureRecognized:(UIPanGestureRecognizer *)recognizer
             CGFloat oppositeScale = (1 - (contentViewScale - 1));
             self.contentViewContainer.transform = CGAffineTransformMakeScale(oppositeScale, oppositeScale);
             self.contentViewContainer.transform = CGAffineTransformTranslate(self.contentViewContainer.transform, point.x, 0);
+            [self updateContentViewAdditionalSafeAreaInsets];
         } else {
             self.contentViewContainer.transform = CGAffineTransformMakeScale(contentViewScale, contentViewScale);
```

Comment:
I have found that scaling `layer.transform` instead of `view.transform` does not cause this bug

Reply:
Great job! Thanks